manager/controllers/utils.go

```diff
@@ -197,6 +197,25 @@ func ApplySearch(c *gin.Context, tx *gorm.DB, searchColumns ...string) *gorm.DB
 	return txWithSearch
 }
 
+// Filter systems by tags,
+// TODO: Implement implicit namespaces
+// TODO: we currently use same encoding inside the DB, as is used in query strings
+// the namespace/name=value encoding. Ensure this is correct by parsing and re-formatting the values
+func ApplyTagsFilter(c *gin.Context, tx *gorm.DB, systemIDExpr string) (*gorm.DB, bool) {
```

Comment:
I don't see the second return val (bool) to be used.

Reply:
It'll be used for switching to manual data calculation in case of `/advisories` endpoint.